ziti/contexts.go

```diff
@@ -0,0 +1,70 @@
+package ziti
+
+import (
+	"github.com/michaelquigley/pfxlog"
+	"github.com/openziti/sdk-golang/ziti/config"
+	"github.com/openziti/sdk-golang/ziti/edge"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+)
+
+var contexts = new(sync.Map)
+
+func init() {
+	ids := os.Getenv("ZITI_IDENTITIES")
```

Comment:
imo. sdk's should do 'magic' with environment variables like this.it should be explicitly initialized